.gitignore

```diff
@@ -1,3 +1,4 @@
+.vagrant
```

Comment:
ha - nice to see it's being used already - does not really belong to this PR, but won't hurt anyone to just add it now, I guess :)